modules/living_spaces_group_privacy/living_spaces_group_privacy.module

```diff
@@ -84,7 +84,26 @@ function living_spaces_group_privacy_entity_base_field_info_alter(&$fields, Enti
  * Implements hook_form_FORM_ID_alter().
  */
 function living_spaces_group_privacy_form_group_form_alter(&$form, FormStateInterface $form_state, $form_id) {
-  $form['living_space_privacy']['#access'] = \Drupal::currentUser()->hasPermission('access living spaces privacy');
+  $user = \Drupal::currentUser();
+  $access = $user->hasPermission('access living spaces privacy');
+
+  $uuid = \Drupal::request()->query->get('space');
+  if (!$access && !empty($uuid)) {
```

Comment:
If there is no space query param you can get space from form, because it's a space edit form.

Reply:
thanks, fixed